applications/ehr_query_llm/lmm/rag/ehr/create_ehr_db.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# BE SURE TO HAVE FHIR APP RUNNING BEFORE STARTING THIS SCRIPT
+
+import logging
+import signal
+import sys
+import time
+from threading import Thread
+from typing import Optional
+
+import zmq
+from langchain.schema.document import Document
+from langchain_community.embeddings import HuggingFaceBgeEmbeddings
+from langchain_community.vectorstores import Chroma
+
+from operators.ehr_query_llm.fhir.ehr_query import FHIRQuery
+from operators.ehr_query_llm.fhir.ehr_response import FHIRQueryResponse
+from operators.ehr_query_llm.message_handling import MessageReceiver
+
+# FHIR global vars
+topic_request = "ehr-request"
+topic_response = "ehr-response"
+busy = False
+resources_to_retrieve = ["Observation", "FamilyMemberHistory", "Condition", "AllergyIntolerance"]
+
+# EHR Patient ID for John Doe
+mrn = "108915cc-b9f2-47db-ad4b-0143151f4f61"
+
+# LLM summary global vars
+ehr_data = None
+
+# Stores EHR requests initiated from this instance
+ehr_request_ids = []
+
+# Vector db global vars
+EHR_FINETUNED_MODEL = "/workspace/volumes/models/bge-large-ehr-finetune"
+CACHE_FOLDER = "/workspace/volumes/models"
+PERSISTENT_FOLDER = "/workspace/holohub/applications/ehr_query_llm/lmm/rag/ehr/db"
+
+
+def get_ehr_data(
+    allow_reqested_only: Optional[bool] = True,
```

Comment:
Typo nitpick (here and below)

```suggestion
    allow_requested_only: Optional[bool] = True,
```